js/core/component_registrator.d.ts

```diff
@@ -21,4 +23,4 @@ export default function registerComponent(name: string, componentClass: any): vo
  * @hidden
  * @prevFileNamespace DevExpress.core
  */
-export default function registerComponent(name: string, namespace: any, componentClass: any): void;
+export default function registerComponent(name: string, namespace: any, componentClass: Component): void;
```

Comment:
`namespace: any` here we should keep `any` since the registrar acts like follows: `namespace[name] = componentClass;`

Reply:
Maybe `object`?